src/test/resources/application.yml

```
@@ -0,0 +1,12 @@
+spring:
+  datasource:
+    url: jdbc:postgresql://localhost:5432/testdb
+    username: testuser
+    password: testpass
+    driver-class-name: org.postgresql.Driver
+  jpa:
+    hibernate:
+      ddl-auto: create-drop   # 테스트 끝나면 테이블 삭제
+    properties:
+      hibernate:
+        dialect: org.hibernate.dialect.PostgreSQLDialect
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)